lib/iris/tests/system_test.py

```diff
@@ -74,7 +74,7 @@ def horiz_cs():
                 new_cube, ("system", "supported_filetype_%s.cml" % filetype)
             )
 
-    def system_test_imports_general(self):
+    def test_imports_general(self):
```

Comment:
`pytest` requires test names to start with "test" or it just won't discover them